include/SugarTheme/getImage.php

```diff
@@ -38,6 +38,9 @@
  * display the words  "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  ********************************************************************************/
 
+if(stristr($_REQUEST['imageName'], 'attachment') !== FALSE) {
```

Comment:
Please use lover case `false` as it should match the PSR standards:
see https://www.php-fig.org/psr/psr-2/#25-keywords-and-truefalsenull